artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -1881,7 +1904,8 @@ public int getEncodeSize() {
          BufferHelper.sizeOfNullableInteger(idCacheSize) +
          BufferHelper.sizeOfNullableSimpleString(pageFullMessagePolicy != null ? pageFullMessagePolicy.toString() : null) +
          BufferHelper.sizeOfNullableInteger(prefetchPageBytes) +
-         BufferHelper.sizeOfNullableInteger(prefetchPageMessages);
```

Comment:
why? I added a new record for AddressSettings, right?

Reply:
I wouldn't bother about this.. it's not going to be used.. the new record would be used.


Although shouldn't we always enable management for internal addresses?

Reply:
Indeed. I'll remove those changes.